plugins/doc_fragments/auth.py

```diff
@@ -238,4 +250,16 @@ class ModuleDocFragment(object):
           - section: hashi_vault_collection
             key: aws_iam_server_id
             version_added: 1.4.0
+      cert_auth_public_key:
+        env:
+          - name: ANSIBLE_HASHI_VAULT_CERT_AUTH_PUBLIC_KEY
```

Comment:
Hi @briantist
would it be possible to add an option to set the `cert_auth_public_key` parameter using the variables `ansible_hashi_vault_cert_auth_public_key` and `ansible_hashi_vault_cert_auth_private_key`?

AWX does not allow setting environment variables with the `ANSIBLE_` prefix, which makes it hard to use the plugin

Reply:
@RoeeFabrikantSAP please open a new issue

Reply:
Thank you!
I have created https://github.com/ansible-collections/community.hashi_vault/issues/428